scene/resources/resource_format_text.cpp

```diff
@@ -900,6 +902,9 @@ void ResourceLoaderText::get_dependencies(Ref<FileAccess> p_f, List<String> *p_d
 		if (p_add_types) {
 			path += "::" + type;
 		}
+		if (!fallback_path.is_empty()) {
+			path += "::" + fallback_path;
+		}
```

Comment:
How will the loader differentiate between `type` and `fallback_path` when reading what comes after `::`, in case both `p_add_types` on/off and non-empty `fallback_path` can happen?

Reply:
I guess there is no way to differentiate that 🙃
But given how `fallback_path` is used, it's not a big problem if it's invalid. Maybe I should add more comments about its caveats? It doesn't need to be super robust for now, as it does its job.

Reply:
Ok I changed it to just add another `::` if `p_add_types` is false, so type and fallback are at fixed positions and can be retrieved reliably.